### ISSUE TITLE
bergfreunde.de: breakage

### ISSUE THREAD

**Issue by divide29** [uBlockOrigin/uAssets#22495]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.bergfreunde.de/
```

### Description

Hi,

i can't see the trusted shops widget on all websites, because their getting blocked

Error Console:  Failed to load resource: net::ERR_BLOCKED_BY_CLIENT

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/92187331/c3eb61a2-c05f-47df-9e3b-78ffddc43f9e)

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.55.0
Chromium: 121
filterset (summary):
 network: 125506
 cosmetic: 50391
 scriptlet: 20445
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  DEU-0: 7258-0, 18h.46m
  easylist: 81111-21, 1h.56m Δ
  easyprivacy: 50202-47, 1h.56m Δ
  plowe-0: 3785-1187, 8d.13h.25m
  ublock-badware: 7861-141, 1h.56m Δ
  ublock-filters: 37288-365, 1h.56m Δ
  ublock-privacy: 976-6, 1h.56m Δ
  ublock-quick-fixes: 123-6, 1h.56m Δ
  ublock-unbreak: 2211-34, 1h.56m Δ
  urlhaus-1: 7485-0, 18h.45m
filterset (user): [empty]
trustedset:
 added: [array of 24 redacted]
switchRuleset:
 added: [array of 3 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 329 ms (selfie)
 maxAssetCacheWait: 127 ms
 cacheBackend: browser.storage.local
popupPanel:
 blocked: 12
 network:
  bing.com: 1
  criteo.com: 1
  dwin1.com: 1
  google.com: 1
  googletagmanager.com: 1
  newrelic.com: 1
  nr-data.net: 1
  optimizely.com: 3
  trustedshops.com: 1
  wt-safetag.com: 1

```
</details>

**Comment by ButterVsMargarine** [comment]
Blocked by EasyPrivacy

**Comment by gorhill** [comment]
Broken by `||widgets.trustedshops.com^$3p` in EasyPrivacy.

cc @ryanbr

**Comment by ButterVsMargarine** [comment]
I remember a similar issue. It didn't get unblocked.

**Comment by ryanbr** [comment]
Intentially blocking the widget, adds trackers.

**Comment by gorhill** [comment]
There is already a 1st party-hosted "trustedshop" icon in there; people can click for more details. Closing as "wontfix" as per @ryanbr's comment.
